contracts/DistributionCreator.sol

```diff
@@ -204,142 +215,140 @@ contract DistributionCreator is UUPSHelper, ReentrancyGuardUpgradeable {
-    /// @dev Users may either call `acceptConditions` here or `sign` the message
+    /// @notice Allows a user to accept Merkl's terms and conditions to enable campaign creation
+    /// @dev Sets the sender's whitelist status to bypass signature requirements
     function acceptConditions() external {
         userSignatureWhitelist[msg.sender] = 1;
     }
 
-    /// @notice Checks whether the `msg.sender`'s `signature` is compatible with the message
-    /// to sign and stores the signature
-    /// @dev If you signed the message once, and the message has not been modified, then you do not
-    /// need to sign again
-    function sign(bytes calldata signature) external {
-        _sign(signature);
-    }
-
-    /// @notice Combines signing the message and creating a campaign
-    function signAndCreateCampaign(
-        CampaignParameters memory newCampaign,
-        bytes calldata signature
-    ) external returns (bytes32) {
-        _sign(signature);
-        return _createCampaign(newCampaign);
-    }
-
-    /// @notice Creates a `distribution` to incentivize a given pool for a specific period of time
-    function createDistribution(
-        DistributionParameters memory newDistribution
-    ) external nonReentrant hasSigned returns (uint256 distributionAmount) {
-        return _createDistribution(newDistribution);
-    }
-
-    /// @notice Same as the function above but for multiple distributions at once
-    function createDistributions(
-        DistributionParameters[] memory distributions
-    ) external nonReentrant hasSigned returns (uint256[] memory) {
-        uint256 distributionsLength = distributions.length;
-        uint256[] memory distributionAmounts = new uint256[](distributionsLength);
-        for (uint256 i; i < distributionsLength; ) {
-            distributionAmounts[i] = _createDistribution(distributions[i]);
-            unchecked {
-                ++i;
-            }
-        }
-        return distributionAmounts;
-    }
-
-    /// @notice Overrides a campaign with new parameters
-    /// @dev Some overrides maybe incorrect, but their correctness cannot be checked onchain. It is up to the Merkl
-    /// engine to check the validity of the override. If the override is invalid, then the first campaign details
-    /// will still apply.
-    /// @dev Some fields in the new campaign parameters will be disregarded anyway (like the amount)
+    /// @notice Updates parameters of an existing campaign while preserving core immutable fields
+    /// @param _campaignId ID of the campaign to override
+    /// @param newCampaign New campaign parameters (some fields will be ignored or validated)
+    /// @dev Cannot change rewardToken, amount, or creator address
+    /// @dev Can only update startTimestamp if the campaign has not yet started
+    /// @dev New end time (startTimestamp + duration) must be in the future
+    /// @dev The Merkl engine validates override correctness; invalid overrides are ignored
```

Comment:
Add a comment to clarify that it may not be processed and that fees would be taken in the engine